styles/docs.rs/catppuccin.user.css

```diff
@@ -15,7 +15,7 @@
 @var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
 ==/UserStyle== */
 
-@-moz-document domain('docs.rs') {
+@-moz-document regexp('^https?://(docs\\.rs|doc\\.rust-lang\\.org).*') {
```

Comment:
Please add a separate `domain()` entry for the new domain.

Reply:
Done